### ISSUE TITLE
Autocomplete popups not announced by screen readers

### ISSUE THREAD

**Issue by gtritchie** [rstudio/rstudio#6167]
### System details

    RStudio Edition : Desktop or Server
    RStudio Version : 1.3.815
    OS Version      : Any
    R Version       : Any

### Steps to reproduce the problem
- with a screen reader running and screen-reader mode enabled, type "ca" in console, or "ca" in an R script in editor, and hit Tab key to invoke autocomplete
- use arrow keys to move up and down through options

### Describe the problem in detail
- on Safari/VoiceOver on Mac, nothing is said, on Chrome/NVDA on Windows, "ca" is read over and over as you move around the autocomplete list

### Describe the behavior you expected
- autocomplete is usable via a screen reader

**Comment by jmcphers** [comment]
New feature; not for 1.3.

**Comment by gtritchie** [comment]
Moving to 1.5; too risky to make these changes at this stage (involves changes to Ace).

**Comment by gtritchie** [comment]
If we get in one screen reader fix for 1.5 (beyond moving to Electron), this would be the highest impact so leaving in 1.5.

**Comment by mikebessuille** [comment]
Moving back to 1.5; it was moved out in a bulk move.

**Comment by gtritchie** [comment]
Makes me sad to move out, again, but more important for accessibility to get Electron port completed, then can tackle this (and benefit both server and desktop users). Moving to Prairie Trillium.

**Comment by gtritchie** [comment]
There has been some work on this on the Ace side (I haven't played with it yet): https://github.com/ajaxorg/ace/pull/5008
